components/Statistics/TourStats.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { dateFormat, DATE, getTotalMillisecondsString } from "@/utils/dateUtil";
+import { Icon, Text, ThemeType, useTheme } from "@ui-kitten/components";
+import { Tour, Stage } from "@/model/model";
+import {
+  getTourDuration,
+  getTourDistance,
+  getTourAverageSpeed,
+} from "@/services/statisticsService";
+
+type TourStatsProps = {
+  tour: Tour;
+};
+
+export default function TourStats(props: TourStatsProps) {
+  const theme = useTheme();
+  const styles = makeStyles(theme);
+  const [stages, setStages] = useState<Stage[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const getStages = await props.tour.stages.fetch();
+      if (getStages) {
+        setStages(getStages);
+      }
+    })();
+  }, [props.tour]);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.stat_column}>
+        <View style={styles.stat_row}>
+          <Icon name="calendar-plus" style={styles.icon_style} />
+          <Text>
+            {props.tour.startedAt
+              ? dateFormat(props.tour.startedAt, DATE)
+              : "--"}
+          </Text>
+        </View>
+        {props.tour.finishedAt && (
+          <View style={styles.stat_row}>
+            <Icon name="calendar-check" style={styles.icon_style} />
+            <Text>{dateFormat(props.tour.finishedAt, DATE)}</Text>
+          </View>
+        )}
+        <View style={styles.stat_row}>
+          <Icon name="clock" style={styles.icon_style} />
+          <Text>{getTotalMillisecondsString(getTourDuration(stages))}</Text>
+        </View>
+      </View>
+      <View style={styles.stat_column}>
+        <View style={styles.stat_row}>
+          <Icon name="arrows-left-right" style={styles.icon_style} />
+          <Text>{getTourDistance(stages).toFixed(2) + " km"}</Text>
+        </View>
+        <View style={styles.stat_row}>
+          <Icon name="arrow-up-right-dots" style={styles.icon_style} />
+          <Text>{"0 m"}</Text>
+        </View>
+        <View style={styles.stat_row}>
+          <Icon name="gauge-high" style={styles.icon_style} />
+          <Text>{getTourAverageSpeed(stages).toFixed(1) + " km/h"}</Text>
+        </View>
+        <View style={styles.stat_row}>
+          <Icon name="bolt" style={styles.icon_style} />
+          <Text>{0 + " kcal"}</Text>
+        </View>
+      </View>
+    </View>
+  );
+}
+
+const makeStyles = (theme: ThemeType): any => {
+  return StyleSheet.create({
+    container: {
+      flexDirection: "row",
+      backgroundColor: theme["color-primary-transparent-500"],
+      height: "auto",
+      paddingHorizontal: 10,
```

Comment:
```suggestion
      paddingHorizontal: 20,
```
Ist mit 10 ein bisschen zu nah am Rand.